main.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_vpc_peering_connection" "request" {
+  provider = "aws.source"
+
+  auto_accept   = "false"
+  peer_owner_id = "${var.vpc_dest_account_id}"
+  peer_vpc_id   = "${var.vpc_dest_vpc_id}"
+  vpc_id        = "${var.vpc_source_vpc_id}"
+
+  tags {
+    Side = "Requester"
+  }
+
+  tags = {
+    Name = "${format("%s - %s", var.vpc_source_name, var.vpc_dest_name)}"
```

Comment:
you should be able to get the name with `data`

```hcl
data "aws_vpc" "dest" {
  id = "${var. vpc_dest_vpc_id}"
}

data "aws_vpc" "source" {
  id = "${var. vpc_source_vpc_id}"
}

// then:
  Name = "${format("%s - %s", data. aws_vpc.source.name, data. aws_vpc.dest.name)}"
```

Reply:
data provider does not seem to pull in region from root hence test is failing with error: "provider.aws: "region": required field is not set"

Reply:
Just pushed a fix for that

Reply:
the issue was you not defining a provider block in the module which was the same gotcha we had before when everything deployed into the CI account